frontend/src/App.js

```diff
@@ -1,12 +1,123 @@
-import React from "react";
-import { Stage, Layer, Rect, Text } from "react-konva";
-import "./App.css";
+import React, { useState } from "react";
+import { Stage, Layer, Line, Circle } from "react-konva";
 
 export default function App() {
   return (
     <>
       <h1>Intersecție de poligoane</h1>
-      <Stage width="360" height="360"></Stage>
+      <PolygonDisplay />
     </>
   );
 }
+
+const initialPolygons = [
+  {
+    id: 0,
+    vertices: [
+      {
+        x: 23,
+        y: 20,
+      },
+      {
+        x: 23,
+        y: 160,
+      },
+      {
+        x: 70,
+        y: 93,
+      },
+      {
+        x: 150,
+        y: 109,
+      },
+    ],
+  },
+];
+
+function PolygonDisplay() {
+  const [polygonData, setPolygonData] = useState(initialPolygons);
+
+  console.log("redrawing polygon display");
+
+  const updatePosition = (polygonIndex, vertexIndex, x, y) => {
+    const dataCopy = JSON.parse(JSON.stringify(polygonData));
+    dataCopy[polygonIndex].vertices[vertexIndex] = { x, y };
```

Comment:
There should be no space after '{' and before '}' in "{ x, y }", but due to the fact that you used it everywhere (your code has consistency), it's all right

Reply:
This is the default formatting used by [Prettier](https://prettier.io/)